operator/src/test/java/oracle/kubernetes/operator/DomainProcessorTest.java

```diff
@@ -521,6 +520,25 @@ public void afterChangeTriggersIntrospection_doesNotRunIntrospectionOnNextExplic
     assertThat(job, nullValue());
   }
 
+  @Test
+  public void whenDomainTypeIsFromModelDomainAndAdminServerModified_runIntrospectionJobFirst2() throws Exception {
```

Comment:
There are two things being checked here; please split it into two tests. Use blank lines to separate setup, execution, and validation.

Reply:
The test tries to simulate the first introspection that returns an error in the domain status and then in the next retry will stop the introspection retries.   How to make it into two tests and the second one runs after the first one without affecting all other tests?